github/orgs_teams.go

```diff
@@ -114,10 +114,34 @@ func (s *OrganizationsService) GetTeam(ctx context.Context, team int) (*Team, *R
 	return t, resp, nil
 }
 
+// NewTeam represents a team to be created or modified.
+type NewTeam struct {
+	ID           *int      `json:"id,omitempty"`
+	Name         *string   `json:"name,omitempty"`
+	Description  *string   `json:"description,omitempty"`
+	Maintainers  []*string `json:"maintainers,omitempty"`
+	RepoNames    []*string `json:"repo_names,omitempty"`
```

Comment:
These two fields can be simplified to `[]string`.

You can look at other structures for example. We use pointers to basic types so it’s possible to tell if the field was missing or zero. Slices are already pointer types and can be nil if missing.